docs/Ariba-batchsource.md

```diff
@@ -23,21 +23,21 @@ You can configure the following properties for the Ariba.
 
 **Reference Name:** Name used to uniquely identify this source for lineage,
 annotating metadata, etc.  
-**API Endpoint (M)**: Ariba Base URL.  
-**System Type**: Select system either Prod or Sandbox.  
-**Realm (M)**: Realm name of the customer.   
-**View Template Name (M)**: Name of the Template.
+**API Endpoint (M)**: Specify the base Path of Ariba API, for example, https://sandbox.api.sap.com.  
```

Comment:
We usually just give the definition: "Base path of the Ariba API, for example....."

Reply:
Changed